### ISSUE TITLE
Use System Proxy

### ISSUE THREAD

**Issue by Riajyuu** [oott123/proxy-proxy#1]
If you don't know which API in bound for, here it is:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/proxy/settings

**Comment by oott123** [comment]
No, proxy-proxy never uses or modify your browser settings. Instead, proxy-proxy listens on [proxy.onRequest](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/proxy/onRequest) event, which handlers should return a [proxy.ProxyInfo](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/proxy/ProxyInfo) object. Unfortunately, you can not use system proxy when using this API.

If extensions wants to use proxy settings to set proxy settings, they can't use onRequest API with it, which means you can only switch between "completely use system proxy" or "use rules but no system proxy option". If that's what you want, you can simply disable proxy-proxy.
